src/main/java/net/sf/jabref/openoffice/CitationManager.java

```diff
@@ -140,20 +140,20 @@ public void showDialog() {
     static class CitEntry implements Comparable<CitEntry> {
 
         private final String refMarkName;
-        private String pageInfo;
+        private Optional<String> pageInfo;
         private final String context;
-        private final String origPageInfo;
+        private final Optional<String> origPageInfo;
 
 
-        public CitEntry(String refMarkName, String context, String pageInfo) {
+        public CitEntry(String refMarkName, String context, Optional<String> optional) {
```

Comment:
I think this is no good use of Optional. This should be done by overlaoding the method.